src/index.js

```diff
@@ -86,7 +87,9 @@ class Dropzone extends React.Component {
     e.preventDefault();
 
     // Only deactivate once the dropzone and all children was left.
-    if (--this.enterCounter > 0) {
+    this.dragTargets = this.dragTargets.filter(el => el !== e.target);
+    this.dragTargets = this.dragTargets.filter(el => this.node.contains(el));
```

Comment:
Also here. This is very confusing to see 2 filters in a row with results overwriting each other. This can be rewritten like:

```
this.dragTargets =  this.dragTargets.filter(el => el !== e.target && this.node.contains(el));
```

but I'm wondering why do even need to assign it back to this instance variable if you're only concerned about the length of the array?

Wouldn't it easier to use `Array.prototype.some` or `Array.prototype.any` instead and not mutate it?